src/algos/graph_search/bfs_and_dfs/in_class/Notes.MD

```diff
@@ -0,0 +1,3 @@
+Queues are used to implement BFS (iteratively)
+
+Stacks are used to implement DFS (recursively)
```

Comment:
While you're not wrong (recursion does use a stack, i.e. the "function call stack"), if you're manually using a stack yourself then you're probably implementing the DFS *iteratively* as well.